lib/orderbook/OrderBookRepository.ts

```diff
@@ -51,6 +51,14 @@ class OrderbookRepository {
   public addTrade = (trade: db.TradeFactory) => {
     return this.models.Trade.create(trade);
   }
+
+  public getTrades = (): Bluebird<db.TradeInstance[]> => {
+    return this.models.Trade.findAll();
+  }
+
+  public getTradesLimit = (limit: number): Bluebird<db.TradeInstance[]> => {
+    return this.models.Trade.findAll({ limit });
```

Comment:
I'm not sure this is returning the most recent trades by default, we may want to order results by `createdAt`.